app/views/traces/index.html.erb

```diff
@@ -17,33 +17,33 @@
     <% if @target_user.blank? %>
       <!-- public traces -->
       <li class="nav-item">
-        <%= link_to t(".all_traces"), { :controller => "traces", :action => "index", :display_name => nil, :page => nil }, { :class => "nav-link active" } %>
+        <%= link_to t(".all_traces"), { :controller => "traces", :action => "index", :display_name => nil, :max_id => nil }, { :class => "nav-link active" } %>
       </li>
       <% if current_user %>
         <li class="nav-item">
-          <%= link_to t(".my_traces"), { :action => "mine", :page => nil }, { :class => "nav-link" } %>
+          <%= link_to t(".my_traces"), { :action => "mine", :max_id => nil }, { :class => "nav-link" } %>
         </li>
       <% end %>
     <% elsif current_user && current_user == @target_user %>
       <li class="nav-item">
-        <%= link_to t(".all_traces"), { :controller => "traces", :action => "index", :display_name => nil, :page => nil }, { :class => "nav-link" } %>
+        <%= link_to t(".all_traces"), { :controller => "traces", :action => "index", :display_name => nil, :max_id => nil }, { :class => "nav-link" } %>
       </li>
       <!-- my traces -->
       <li class="nav-item">
-        <%= link_to t(".my_traces"), { :action => "mine", :page => nil }, { :class => "nav-link active" } %>
+        <%= link_to t(".my_traces"), { :action => "mine", :max_id => nil }, { :class => "nav-link active" } %>
```

Comment:
I don't think any of these `:max_id => nil` are needed, since the link is being built from scratch (other links are built from merging with existing params elsewhere).

Or if I've misunderstood, and they are necessary, then I think a `:min_id => nil` would be needed too.